src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Tables/OrderStatus.sql

```diff
@@ -1,5 +1,8 @@
 ﻿CREATE TABLE dbo.OrderStatus
 (
-    OrderStatusId INT IDENTITY(1,1) PRIMARY KEY, 
-    [Name] NVARCHAR(30) NULL
+    OrderStatusId INT IDENTITY(1,1), 
+    [Name] NVARCHAR(30) NULL,
+
+    CONSTRAINT PK_OrderStatusId PRIMARY KEY (OrderStatusId)
+

```

Comment:
I can't believe you added a new line here, RIP

Reply:
Sad times 😢